packages/vite/src/node/optimizer/index.ts

```diff
@@ -253,6 +253,8 @@ export function loadCachedDepOptimizationMetadata(
       // need to resolve the processing promise so requests can move on
       return cachedMetadata
     }
+  } else {
+    config.logger.info('Forced re-optimization of dependencies')
```

Comment:
```suggestion
    log('Forced re-optimization of dependencies')
```

Should this be a `log` or is it intentionally using the logger?

Reply:
It is intentional, we want this log when `asCommand` is false too